docs/contributing.md

```diff
@@ -26,7 +26,7 @@ accept your pull requests.
 ## Building Projects
 
 There are a couple of commands that will help you make working with these
-modules a little easier:
+libraries a little easier:
```

Comment:
I think `libraries` is more beginner friendly, but modules is more correct. Below we have a number of packages which are technically libraries and then other things that are very much not (e.g `sw-cli` is a CLI). 

I'd be in favor of renaming things currently called modules to `libraries` if we aren't doing it for packages that blur those lines (sw-build) or are outright not libraries (cli).